flipperdevices/flipperzero-firmware
# [FL-2520] FW build with -Wextra

# What's new

- [FL-2520] Tons of changes fixing various compiler warnings produced by -Wextra

# Verification 

- Build firmware 
- Ensure that build passes -Wextra to compiler args and no errors are produced
- Check all features to be working 

# Checklist (For Reviewer)



[FL-2520]: https://flipperzero.atlassian.net/browse/FL-2520?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
